docs/prose/conf.py

```diff
@@ -56,8 +59,9 @@
     'myst_parser',
 ]
 
+apidocsbasepath = ('https://' + os.environ.get('VERCEL_URL') + '/api/api%s', '') if 'VERCEL_URL' in os.environ else ('https://docs.inrupt.com/client-libraries/api/js/solid-client%s','')
```

Comment:
Related to the other question: if the API docs go into an API subdirectory, then I think this would need to be:

```suggestion
apidocsbasepath = ('https://' + os.environ.get('VERCEL_URL') + '/api/api%s', '') if 'VERCEL_URL' in os.environ else ('https://docs.inrupt.com/client-libraries/api/js/solid-client/api%s','')
```

Additionally, these links do not include the `.html` extension, whereas Sphinx _does_ seem to add them? To be consistent, we could use them as well, as in 76bc593a5de29837e425c083dd68db0c289466ae?

Reply:
Ah - you are absolutely correct.  And it probably shouldn't go in the api directory - to avoid repetition of api in the site structure.